astropy/io/fits/convenience.py

```diff
@@ -709,6 +710,105 @@ def info(filename, output=None, **kwargs):
     return ret
 
 
+def printdiff(inputa, inputb, *args, **kwargs):
+    """
+    Compare two parts of a fits file, including entire FITS files, FITS HDULists and
+    FITS HDU objects.
+
+    Parameters
+    ----------
+    inputa : str of file path, HDUList object, or HDU object
+        The filename of a FITS file on disk, or HDUList or HDU
+        object to compare to inputb.
+
+    inputb : str of file path, HDUList object, or HDU object
+        The filename of a FITS file on disk, or HDUList or HDU
+        object to compare to inputa.
+
+    ext, extname, extver
+        Additional positional arguments are for extension specification if your
+        inputA and inputB are string filenames (will not work if inputA and
```

Comment:
Given that Python is case-sensitive, change `inputA` to `inputa` to be consistent with above; same applies to `inputB`.